### DISCUSSION TITLE
Why is my Shopify Integration disabled?

### DISCUSSION THREAD

**Discussion by Giszmo** [btcpayserver/btcpayserver#2532] [Technical Support] [answered]
I'm struggling with the Shopify integration.

Shopify made me Pinky-promise to to not mess with the checkout process. This step is not covered in [the documentation](https://docs.btcpayserver.org/Shopify/).

Then on the btcpayserver side I follow the steps but after pasting the "example url":

![Screenshot from 2021-05-18 19-46-24](https://user-images.githubusercontent.com/96378/118738680-96472100-b815-11eb-8ccf-faddb6612246.png)

Shopify remains disabled:

![Screenshot from 2021-05-18 19-46-38](https://user-images.githubusercontent.com/96378/118738695-a101b600-b815-11eb-9e87-4d7586f38ca5.png)

I can't find any general setting to enable it and the "disabled" also is not clickable to get more information.

Where do I go from here?

**Comment by NicolasDorier** [comment]
We'll look into it. That said, it is better you create an issue rather than a discussion so we can put it on our milestone for next release.
We might do quick special release just for bug fixes like this.

**Reply by Giszmo** [reply]
Well, I wasn't sure if I missed some switch to activate/deactivate plugins or something and the "new issue" template is really going to great length to avoid issues in favor of faq, support chat, etc. Will create an issue ...

**Comment by NicolasDorier** [comment, marked as answer]
Will be fixed in 1.1.2
